dockerfiles/remote-plugin-go-1.10.7/Dockerfile

```diff
@@ -67,10 +67,11 @@ RUN set -eux; \
     ; \
 	apk del .build-deps \
     ; \
-	mkdir ${HOME}/.cache && chmod -R 777 ${HOME}/.cache \
+	mkdir /.cache && chmod -R 777 /.cache \
 	; \
-    go version
+    go version && \
+    apk add gcc g++
```

Comment:
What do we need gcc?

Reply:
![image](https://user-images.githubusercontent.com/5337267/54681641-8d634380-4b15-11e9-9578-864f9577b372.png)

Probably not every project but it is not uncommon for a go build to depend on gcc and g++

Reply:
I think these dependencies are not a problem. There are some projects which needed it. So for me it's looks ok.